assets/src/styles/blocks/core-overrides/_media-text.scss

```diff
@@ -0,0 +1,35 @@
+.wp-block-media-text {
+  @media (max-width: 600px) {
+    .wp-block-media-text__content {
+      padding: 0;
+      margin-top: $sp-2;
+    }
+
+    .wp-block-button,
+    .wp-block-button a {
+      width: 100%;
+    }
+  }
+
+  @media (min-width: 601px) {
+    &.has-media-on-the-right .wp-block-media-text__content {
+      padding-inline-start: 0;
```

Comment:
Why not `$sp-0`? :grin:

Reply:
I'm pretty sure 0 will remain 0 if we decide to change the grid in the future 😉

Reply:
It makes equally much sense to me as with other numbers :shrug: